.travis.yml

```diff
@@ -16,3 +16,8 @@ matrix:
       script:
         - cargo build
         - cargo build --features small
+    - rust: nightly
+      name: Miri
+      script:
+        - rustup component add miri || travis_terminate 0
+        - cargo miri test
```

Comment:
Great to see Miri being added here. :)

`cargo miri test` will interactively ask for confirmation to install things, and will probably not work. Miri is just not available for the latest nightly so we never enter this code path.
The recommended Miri CI integration is [given in our README](https://github.com/rust-lang/miri/#running-miri-on-ci).